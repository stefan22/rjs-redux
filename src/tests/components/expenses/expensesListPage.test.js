import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import ExpensesListPage from '../../../router/pages/expenses-list-page';

describe('Expenses-list page', () => {
  it('should render expenses-list page correctly', () => {
    const wrapper = shallow(<ExpensesListPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });

  it('should contain `h1` of `Expenses List page`', () => {
    const wrapper = shallow(<ExpensesListPage />);
    expect(wrapper.find('h1').text()).toBe('Expenses List page');
  });

});
