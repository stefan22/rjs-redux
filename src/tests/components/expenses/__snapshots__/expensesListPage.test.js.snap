// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expenses-list page should render expenses-list page correctly 1`] = `
<div
  className="container"
>
  <header
    className="header expenses-list"
  >
    <h1
      className="text-center"
    >
      Expenses List page
    </h1>
    <p
      className="text-center"
    >
      (filters off description)
    </p>
  </header>
  <div
    className="main-content"
  >
    <Connect(ExpensesListFilters) />
    <Connect(ExpensesList) />
  </div>
</div>
`;
